packages/styles/scss/components/leadspace/_leadspace.scss

```diff
@@ -153,6 +157,21 @@ $btn-min-width: 26;
       flex-flow: column nowrap;
     }
 
+    .#{$prefix}--leadspace__action {
+      display: inline-block;
+
+      // Button requirements dictate buttons should be full width at 320px and
+      // down. The carbon--breakpoint-down(sm) mixin has logic that will use
+      // a value of 320px - 0.02, which is not quite what we want, so instead,
+      // we use carbon--breakpoint-down with a number argument, using the
+      // defined width for the sm breakpoint.
+      $breakpoint: map-get($carbon--grid-breakpoints, sm);
```

Comment:
@proeung @jkaeser Is this an OK approach to you?

Reply:
I'm actually not sure!

I imagine using `carbon--breakpoint-down(sm)` causes this style to apply up to 319px, not 320px, and I wonder if that's actually okay. Assuming the `@carbon/grid` package is compliant with the design spec, perhaps this is the designed behavior and it's unwise to deviate from it.

On the other hand, I trust the designers understand the intricacies of the design system way better than me, and their specs (below) should be followed to the letter.

<img width="509" alt="Screen Shot 2023-02-20 at 5 00 12 PM" src="https://user-images.githubusercontent.com/12532727/220204637-5492f76a-bfa6-4af0-857b-02610268cdc3.png">

Reply:
> I imagine using carbon--breakpoint-down(sm) causes this style to apply up to 319px, not 320px

Yes, that's the effect in practice when I tried it locally.

> On the other hand, I trust the designers understand the intricacies of the design system way better than me, and their specs (below) should be followed to the letter.

I agree, in which case, this was the only option I could come up with. It feels safe to me, we would inherit any changes to the default set of breakpoints in the same way we would if we were able to use the mixins directly.